src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    addBook: (state, action) => {
+      const newId = state.length === 0 ? 1 : state[state.length - 1].id + 1;
+      const newBook = {
+        id: newId,
+        // id: Date.now(),
```

Comment:
- [ ] You have done great adding the `booksSlice` and the reducers. :+1: Following the best practices, you are not meant to leave unused code as a comment, it makes your codebase look a bit unprofessional. I'll suggest you remove that. :+1: :100: